phivk/gradu.al
# Feature/most popular nodes

To test:
- Navigate to /mozfest
- Check that the list component doesn't appear on large screens
- Check that the graph component is swapped for the list component on mobile screens
